package.json

```diff
@@ -1,20 +1,17 @@
 {
   "name": "language-pegjs",
   "version": "0.5.0",
-  "description": "PEG.js language support in Atom",
+  "description": "PEG.js language support in Pulsar",
   "engines": {
     "atom": "*",
     "node": "*"
   },
-  "homepage": "http://atom.github.io/language-pegjs",
+  "homepage": "https://github.com/pulsar-edit/language-pegjs",
   "repository": {
     "type": "git",
-    "url": "https://github.com/atom/language-pegjs.git"
+    "url": "https://github.com/pulsar-edit/language-pegjs.git"
   },
   "bugs": {
-    "url": "https://github.com/atom/language-pegjs/issues"
-  },
-  "devDependencies": {
-    "coffeelint": "^1.10.1"
+    "url": "https://github.com/pulsar-edit/language-pegjs/issues"
```

Comment:
To make things more in line with other `package.json`s I would suggest removing the `"homepage"` and `"bugs"` values and change the `"repository"` value to `https://github.com/pulsar-edit/language-pegjs`.

Reply:
All fantastic ideas, lemme address those. Thanks a ton for the review @Sertonix! Always appreciate your input, and love to see you pop up in my notifications again

Reply:
I hope to get a bit more active soon.